src/PostgREST/Auth.hs

```diff
@@ -39,7 +39,7 @@ data JWTAttempt = JWTInvalid JWTError
   of JWT claims.
 -}
 jwtClaims :: Maybe JWK -> Maybe StringOrURI -> LByteString  -> IO JWTAttempt
-jwtClaims _ Nothing "" = return $ JWTClaims M.empty
+jwtClaims _ _ "" = return $ JWTClaims M.empty
```

Comment:
Given that you now turn `Just ""` into `Nothing` in the config parser, does this pattern match need to be updated too?

Reply:
it would probably work without this change but on it's own `jwtClaims` does not seem right the way it was. The `M.empty` result is determined by the presence (or absence) of the JWT, the `aud` config value should not influence that.

Reply:
If I am not mistaken this is actually changing a very old behavior, isn't ?